ibl_to_nwb/AlyxToNWB/alyx_to_nwb_converter.py

```diff
@@ -388,15 +396,27 @@ def create_acquisition(self):
                 else:
                     if i['name'] in ['raw.lf', 'raw.ap']:
                         for j, probes in enumerate(range(self.no_probes)):
```

Comment:
`i` and `j`, etc. are ok for small for loops where the meaning if obvious, but they are confusing for large ones like this. Can you please rename these countervariables to things like `dataset_num` `probe_num` or something like that?

Reply:
i'll merge this and create a separate pr for renaming and adding comments where it would help